tests/docs-accuracy.test.sh

```
@@ -717,6 +723,317 @@ for dead_backend_witness in \
+# spawned agent" without naming the scope that makes it true. On the default
+# `workflow` backend there is no per-child argv at all.
+# CHANGELOG.md and docs/ are exempt: append-only history and RFC prose describe
+# the world as it was when written.
+auto_unscoped_hits=""
+while IFS= read -r shipped_md; do
+  case "$shipped_md" in
+    ./CHANGELOG.md|./docs/*|./.git/*|*/node_modules/*) continue ;;
+  esac
+  while IFS= read -r auto_line; do
+    [ -n "$auto_line" ] || continue
+    auto_line_stripped="$(sed 's/--auto-mode//g' <<<"$auto_line")"
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)